### ISSUE TITLE
Uninitialized string offset 0

### ISSUE THREAD

**Issue by webconstructor** [methnen/m-chart#189]
We get lots of warnings in our logs:
`Uninitialized string offset 0 in /srv/wordpress/wp-content/plugins/m-chart/components/class-m-chart-chartjs.php on line 169.`
about:
[https://github.com/methnen/m-chart/blob/master/components/class-m-chart-chartjs.php#L169](url)

Could this be programmed a little more defensively?

**Comment by methnen** [comment]
Yeah, that's funny. So basically there's no chart data yet and it's throwing the warning. That's an easy enough fix.

I'll try to get that out quickly. That's an easy .x.x release.

**Comment by webconstructor** [comment]
Nice, thx

**Comment by methnen** [comment]
Spent a little time on this and I oddly can't duplicate it at all. That said, I've added an additional check to make sure there's something there before passing it to the parse function.

I also fixed another warning I found along the way.

I you could try this one out and see if it works that would be awesome and I'll then push it live to everyone. It's a small change.

**Comment by webconstructor** [comment]
Hi Jamie

We only have this on our production servers. So if you have programmed this a bit more defensively and it doesn’t break anything we can test it with a new release.

greetings Inigo

**Comment by methnen** [comment]
Ah, so the warning is only happening on production. That's interesting. 

It's a small change and it should not break anything.
